src/lingrex/trimming.py

```diff
@@ -0,0 +1,201 @@
+"""
+Trimming functionalities in lingrex.
+"""
+import random
+import collections
+from lingpy.sequence.sound_classes import token2class
+
+
```

Comment:
I always like a `__all__` global to declare what's supposed to be externally used and what functions are considered module-internal, i.e. implemnentation detail.
See https://docs.python.org/3/tutorial/modules.html#importing-from-a-package